akka-actor/src/main/scala/akka/util/OptionVal.scala

```diff
@@ -9,6 +9,7 @@ package akka.util
 private[akka] object OptionVal {
 
   def apply[A >: Null](x: A): OptionVal[A] = new OptionVal(x)
+  def fromNullable[A >: Null](x: A): OptionVal[A] = new OptionVal(x)
```

Comment:
Good addition

(so this is how you convince the compiler to accept Null `>: Null`)